### DISCUSSION TITLE
Data Provider mysql throwing version error

### DISCUSSION THREAD

**Discussion by shreddy99** [drakkan/sftpgo#983] [Q&A]
Hi, I am using MySql DB service from azure. MySQL Version is 8.0. However docker container is throwing below errors.

{"level":"error","time":"2022-09-12T18:38:44.731","sender":"dataprovider_mysql","message":"database version 10 is too old, please see the upgrading docs"}
{"level":"error","time":"2022-09-12T18:38:44.732","sender":"dataprovider_mysql","message":"database migration error: database version 10 is too old, please see the upgrading docs"}

**Comment by shreddy99** [comment]
And i am using docker image version:
drakkan/sftpgo:v2.3.3
DIGEST:sha256:68bd5b18c2fa4c5975616b956554c538896fa4f34603d9b727ab895e56692a61

**Reply by shreddy99** [reply]
Hi @drakkan, when you get a chance can you please look in to this and do the needful.

**Reply by drakkan** [reply]
> Hi @drakkan, when you get a chance can you please look in to this and do the needful.

There is nothing to fix. I'm ignoring you just like you are ignoring the [support policy](https://github.com/drakkan/sftpgo#support-policy), good luck

**Reply by shreddy99** [reply]
Sure, not a problem. I couldn't find any where in the docs that MySQL is not supported for docker image v2.3.3. The same is working for docker image drakkan/sftpgo:v2.2.2. Thought its a bug and raised the issue here. Either ways i will check with my manager to see if we can sponsor. 
Have a good one.

**Reply by drakkan** [reply]
> Sure, not a problem. I couldn't find any where in the docs that MySQL is not supported for docker image v2.3.3. The same is working for docker image drakkan/sftpgo:v2.2.2. Thought its a bug and raised the issue here. Either ways i will check with my manager to see if we can sponsor. Have a good one.

it is supported, the issue is on your side
